static/figure/js/views/right_panel_view.js

```diff
@@ -78,14 +242,11 @@
         events: {
             "submit .new-label-form": "handle_new_label",
             "click .dropdown-menu a": "select_dropdown_option",
-            "click .edit_rois": "editRois",
-        },
-
-        editRois: function(event) {
-            $("#roiModal").modal("show");
+            // "click .edit_rois": "editRois",
```

Comment:
Is there a reason to leave in the commented out event given the function has been removed?